calculator.rb

```diff
@@ -0,0 +1,186 @@
+##### START INITIAL HASHES #####
+
+# store operation, num_1, and num_2 after entry by user
+# important note - sometimes num_1 and num_2 are stored with revised formatting, but sometimes they are not
+user_input = {}
+
+# list all valid operations that can be requested by the user
+valid_operators = {:+ =>["add", "addition", "+"], :- =>["subtract", "subtraction", "-"], :* =>["multiply", "multiplication", "*", "x", "X"], :/ =>["divide", "division", "/"]}
+
+# wip: valid operations for exponential functions and modular functions
+# :** =>["exponent", "exponents", "^", "**"], :% =>["modulo", "modular function", "modularfunction", "%"]
+
+##### END INITIAL HASHES #####
+
+##### START FUNCTIONS #####
+
+# perform addition
+def addition(num_1, num_2)
+  num_1 + num_2
+  return num_1 + num_2
```

Comment:
These two lines are redundant - you should be able to get away with only the second.